openedx/core/djangoapps/schedules/templates/schedules/edx_ace/recurringnudge_day3/email/body.html

```diff
@@ -50,16 +50,38 @@
                     -webkit-border-radius: 4px;
                     -moz-border-radius: 4px;
                     background-color: #005686;
-                    border-top: 10px solid #005686;
-                    border-bottom: 10px solid #005686;
-                    border-right: 16px solid #005686;
-                    border-left: 16px solid #005686;
+                    border-top: 2px solid #005686;
+                    border-bottom: 2px solid #005686;
+                    border-right: 2px solid #005686;
+                    border-left: 2px solid #005686;
                     display: inline-block;
+                    padding: 1rem 5rem;
```

Comment:
nit: probably makes sense to use the same units throughout here, easier to reason about I think.